public/pages/Rules/components/RuleEditor/RuleEditorFormState.model.ts

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Let's just name this file as `RuleEditorFormState.ts`? Adding the model suffix would be a new pattern specific to this plugin and would be confusing.